lib/src/Uncertainty/Algorithm/MetaModel/MetaModelValidation.cxx

```diff
@@ -179,24 +206,39 @@ GridLayout MetaModelValidation::drawValidation() const
 void MetaModelValidation::save(Advocate & adv) const
 {
   PersistentObject::save(adv);
-  adv.saveAttribute( "inputSample_", inputSample_ );
   adv.saveAttribute( "outputSample_", outputSample_ );
-  adv.saveAttribute( "metaModel_", metaModel_ );
-  adv.saveAttribute( "isInitialized_", isInitialized_ );
+  adv.saveAttribute( "metaModelPredictions_", metaModelPredictions_ );
   adv.saveAttribute( "residual_", residual_ );
-  adv.saveAttribute( "q2_", q2_ );
+  adv.saveAttribute( "r2Score_", r2Score_ );
+  adv.saveAttribute( "meanSquaredError_", meanSquaredError_ );
 }
 
 /* Method load() reloads the object from the StorageManager */
 void MetaModelValidation::load(Advocate & adv)
 {
   PersistentObject::load(adv);
-  adv.loadAttribute( "inputSample_", inputSample_ );
   adv.loadAttribute( "outputSample_", outputSample_ );
-  adv.loadAttribute( "metaModel_", metaModel_ );
-  adv.loadAttribute( "isInitialized_", isInitialized_ );
+  if (adv.hasAttribute("metaModelPredictions_"))
+    adv.loadAttribute( "metaModelPredictions_", metaModelPredictions_ );
+  else
+    adv.loadAttribute( "outputSample_", metaModelPredictions_ );
```

Comment:
not really, it should have been metaModel_(outputSample_) with the old attributes

Reply:
You probably mean `metaModel_(inputSample_)`, not . `metaModel_(outputSample_)`. But `outputSample_` is an attribute of the object in OT 1.22, so that the line 224 will work, isn'it? Or do you want me to change this to `metaModel_(inputSample_)`?

Reply:
yes, "metaModel_(inputSample_)", (as the attributes dont exist anymore just load them in temporary variables instead of attibutes)

Reply:
Ok. Fixed at c38c9197a283eb00855942634c7eefbc006a405b.